### ISSUE TITLE
code for initial training and finetune missing?

### ISSUE THREAD

**Issue by vincentme** [lll143653/amsnet#5]
Hi, 

The supplemental materials, you mentioned the initial training with scheduler cosine annealing, then a finetune stage. However, in the current repo I could not find them. If is it possible for you to update the code? Thanks!

**Comment by lll143653** [comment]
Sorry for replying just now, I have modified the code, you can use the following code to fine-tune it

``` shell
python main.py fit --config ./configs/restormer.yaml --model.scheduler.init_args.T_max=25000 --model.optimizer.init_args.lr=0.00001
```
